.github/workflows/ci.yaml

```diff
@@ -47,10 +47,10 @@ jobs:
       - name: Run sqlfluff on jinja templates
         run: |
           sqlfluff lint
-      - name: Run black
+      - name: Run ruff
         if: success() || failure() # still run black if above checks fails
         run: |
-          black --check --verbose .
+          ruff
```

Comment:
Ah... sensible defaults.

I don't think this will verify that the code is formatted as you expect, yeah? (i.e. that `ruff format` was run already -- which it might not be if the dev didn't install commit hooks, which drive-by contributors probably will forget to do). So it doesn't really replace the previous `black` command.

You might want to also add `ruff format --check` here

Reply:
Ah, this is obsolete by the time I submitted. Marking resolved.